File: Latihan B/PYTHON/Pc.py
from Processor import Processor
from Disk import Disk
from Ram import Ram

class Pc:
    
    __pr = Processor()
    __dk = Disk()
    __rm = Ram()
    __totalPrice = 0

    # konstruktor
    def __init__(self, pr = Processor, dk = Disk, rm = Ram):
        self.__pr = pr
        self.__dk = dk
        self.__rm = rm
        self.__totalPrice = pr.getPrice() + dk.getPrice() + rm.getPrice()

    # set semua variable
    def setProc(self, pr):
        self.__pr = pr

    def setDisk(self, dk):
        self.__dk = dk

    def setRam(self, rm):
        self.__rm = rm

    def setTPrice(self, pr = Processor, dk = Disk, rm = Ram):
        self.__totalPrice = pr.getPrice() + dk.getPrice() + rm.getPrice()

    # get untuk semua variable
    def getProc(self):
        return self.__pr
    def getDisk(self):
        return self.__dk
    def getRam(self):
        return self.__rm
    def getTPrice(self):
        return self.__totalPrice
    

    # tampilan untuk pc
    def printPC(self):

        print(" +PROCESSOR")
        self.__pr.printProc()
        print(" +DISK")
        self.__dk.printDisk()
        print(" +RAM")
        self.__rm.printRam()
       

File: Latihan B/PHP/Pc.php
<?php

class PC {
	
	private $pr;
	private $dk;
	private $rm;
	private $totalPrice;

	// Konstruktor
	function __construct($pr, $dk, $rm) {
		$this->pr = $pr;
		$this->dk = $dk;
		$this->rm = $rm;
		$this->totalPrice = $pr->getPrice() + $dk->getPrice() + $rm->getPrice();
	}

	// set untuk semua varible
	function setProc($pr) {
		$this->pr = $pr;
	}
	function setDisk($dk) {
		$this->dk = $dk;
	}
	function setRam($rm) {
		$this->rm = $rm;
	}
	function setTPrice($pr, $dk, $rm) {
		$this->$totalPrice = $pr->getPrice() + $dk->getPrice() + $rm->getPrice();
	}

    //get untuk semua variable
	function getProc() {
		return $this->pr;
	}
	function getDisk() {
		return $this->dk;
	}
	function getRam() {
		return $this->rm;
	}
	function getTPrice() {
		return $this->totalPrice;
	}


    // tampilan untuk semua item didalam pc
    function printPc() {
        echo "+PROCESSOR+". "<br/>". "<br/>";
        $this->pr->printProc();
        echo "<br>". "+DISK+". "<br/>". "<br/>";
        $this->dk->printDisk();
        echo "<br>". "+RAM+". "<br/>". "<br/>";
        $this->rm->printRam();
        echo "------------------------------- +". "<br>";
        echo "Total harga : ". $this->getTPrice(). "<br/>";
    }

    // destructor
    function __destruct() {}
}
?>